tests/glsl-intrinsic/shader-subgroup/shader-subgroup-quad.slang

```diff
@@ -0,0 +1,120 @@
+//TEST:SIMPLE(filecheck=CHECK_GLSL):  -allow-glsl -stage compute -entry computeMain -target glsl
+//TEST:SIMPLE(filecheck=CHECK_SPV):  -allow-glsl -stage compute -entry computeMain -target spirv
+//TEST:SIMPLE(filecheck=CHECK_HLSL): -allow-glsl -stage compute -entry computeMain -target hlsl -DTARGET_HLSL
+
+// not testing cuda due to missing impl
+//T-EST:SIMPLE(filecheck=CHECK_CUDA): -allow-glsl -stage compute -entry computeMain -target cuda -DTARGET_CUDA 
+// not testing cpp due to missing impl
+//T-EST:SIMPLE(filecheck=CHECK_CPP):  -allow-glsl -stage compute -entry computeMain -target cpp
```

Comment:
This test can run for HLSL but it cannot run for CUDA and CPP?
Doesn't HLSL have support for CUDA and CPP?
If it does, you can simply call them or copy and paste whatever HLSL does.

Reply:
the test does not exist for hlsl.meta (the code also does not exist) 

most things have never been tested